ci/cirrusci.sh

```diff
@@ -36,7 +36,12 @@ if [ "$OS_NAME" == "linux" ]; then
 elif [ "$OS_NAME" == "osx" ]; then
   # upgrade GNU make
   brew install make
-  sudo ln -s /usr/local/opt/make/libexec/gnubin/make /usr/local/bin/make
+  for candidateDir in /usr/local/opt/make/libexec/gnubin /opt/homebrew/opt/make/libexec/gnubin; do
+    if [[ -f $candidateDir/make ]]; then
+      sudo ln -s $candidateDir/make /usr/local/bin/make
+      break
+    fi
+  done
```

Comment:
[the location of *GNU* make is different for the arm64 macOS runners (vs. x86_64 macos-13 runners)]